homeassistant/components/osramlightify/light.py

```diff
@@ -4,270 +4,346 @@
 For more details about this platform, please refer to the documentation at
 https://home-assistant.io/components/light.osramlightify/
 """
-from datetime import timedelta
 import logging
 import random
 import socket
 
 import voluptuous as vol
 
-from homeassistant import util
 from homeassistant.components.light import (
-    ATTR_BRIGHTNESS, ATTR_COLOR_TEMP, ATTR_EFFECT, ATTR_HS_COLOR,
-    ATTR_TRANSITION, EFFECT_RANDOM, PLATFORM_SCHEMA, SUPPORT_BRIGHTNESS,
+    ATTR_BRIGHTNESS, ATTR_COLOR_TEMP, ATTR_HS_COLOR, ATTR_TRANSITION,
+    ATTR_EFFECT, EFFECT_RANDOM, PLATFORM_SCHEMA, SUPPORT_BRIGHTNESS,
     SUPPORT_COLOR, SUPPORT_COLOR_TEMP, SUPPORT_EFFECT, SUPPORT_TRANSITION,
     Light)
+
 from homeassistant.const import CONF_HOST
 import homeassistant.helpers.config_validation as cv
-from homeassistant.util.color import (
-    color_temperature_kelvin_to_mired, color_temperature_mired_to_kelvin)
 import homeassistant.util.color as color_util
 
-REQUIREMENTS = ['lightify==1.0.6.1']
+REQUIREMENTS = ['lightify==1.0.7.2']
 
 _LOGGER = logging.getLogger(__name__)
 
 CONF_ALLOW_LIGHTIFY_NODES = 'allow_lightify_nodes'
 CONF_ALLOW_LIGHTIFY_GROUPS = 'allow_lightify_groups'
+CONF_INTERVAL_LIGHTIFY_STATUS = 'interval_lightify_status'
+CONF_INTERVAL_LIGHTIFY_CONF = 'interval_lightify_conf'
 
 DEFAULT_ALLOW_LIGHTIFY_NODES = True
 DEFAULT_ALLOW_LIGHTIFY_GROUPS = True
-
-MIN_TIME_BETWEEN_FORCED_SCANS = timedelta(milliseconds=100)
-MIN_TIME_BETWEEN_SCANS = timedelta(seconds=10)
-
-SUPPORT_OSRAMLIGHTIFY = (SUPPORT_BRIGHTNESS | SUPPORT_COLOR_TEMP |
-                         SUPPORT_EFFECT | SUPPORT_COLOR |
-                         SUPPORT_TRANSITION)
+DEFAULT_INTERVAL_LIGHTIFY_STATUS = 5
+DEFAULT_INTERVAL_LIGHTIFY_CONF = 3600
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_HOST): cv.string,
     vol.Optional(CONF_ALLOW_LIGHTIFY_NODES,
                  default=DEFAULT_ALLOW_LIGHTIFY_NODES): cv.boolean,
     vol.Optional(CONF_ALLOW_LIGHTIFY_GROUPS,
                  default=DEFAULT_ALLOW_LIGHTIFY_GROUPS): cv.boolean,
+    vol.Optional(CONF_INTERVAL_LIGHTIFY_STATUS,
+                 default=DEFAULT_INTERVAL_LIGHTIFY_STATUS): cv.positive_int,
+    vol.Optional(CONF_INTERVAL_LIGHTIFY_CONF,
+                 default=DEFAULT_INTERVAL_LIGHTIFY_CONF): cv.positive_int
 })
 
+DEFAULT_BRIGHTNESS = 2
+DEFAULT_KELVIN = 2700
+
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+def setup_platform(_hass, config, add_entities, _discovery_info=None):
```

Comment:
Why rename the arguments? These are standardized.

Reply:
To make pylint happy and to emphasize that they are not actually used.  But if it's a standard, I can rename them  Should I?

Thanks!

Reply:
Yes, please revert to the previous names in a new PR.

Reply:
Ok!  I'm going to open a new PR with the new Osram Lightify functionality right now, and I will include that change there.

Reply:
See https://github.com/home-assistant/home-assistant/pull/22597.